pkgs/fc/ceph/src/fc/ceph/logs.py

```diff
@@ -50,3 +55,40 @@ def slowreq_histogram(self, include, exclude, first_n, filenames):
             if n >= first_n:
                 break
             n += 1
+
+    def account_s3_traffic(self, state_file: Path, enc_file: Path):
```

Comment:
lets gate the accounting timer behind a feature flag so in case this explodes and we have to adjust things